packages/studio-plugin/src/utils/TypeGuards.ts

```diff
@@ -122,6 +123,8 @@ export default class TypeGuards {
               : !metadata.required
           )
         );
+      case PropValueType.TailwindClass:
+        return typeof value === "string" || value === undefined;
```

Comment:
I don't think we'd always want to allow `undefined` as a valid value. If the prop is required, then the value would need to be a string. IIRC, so far we only use this typeguard for defined values. But, if we need to handle a case where the value is `undefined`, then we'd want to use `propMetadata.required` to determine if that's valid

Reply:
updated! I thought I needed this for some weird undefined button behavior but it seems to be working without it now